inji-web/src/utils/storage.ts

```diff
@@ -1,17 +1,24 @@
+
 export class storage {
+    static SESSION_INFO = "download_session";
+    static SELECTED_LANGUAGE = "selected_language";
 
-    static SESSION_INFO = "download_session"
-    static SELECTED_LANGUAGE = "selected_language"
-    static setItem = (key: string, value: string) => {
-        if (value) {
+    static setItem = (key: string, value: string | null | undefined) => {
+        if (value !== null && value !== undefined) {
             localStorage.setItem(key, JSON.stringify(value));
         }
     }
-    static getItem = (key: string) => {
-        let data = localStorage.getItem(key);
+
+    static getItem = (key: string): string | null => {
+        const data = localStorage.getItem(key);
         if (data) {
-            data = JSON.parse(data);
+            try {
+                return JSON.parse(data);
+            } catch (error) {
+                return null;
+            }
+
         }
-        return data;
+        return null;
```

Comment:
This has to return data, why it is returning null ?

Reply:
this is for the case when invalide  json data is entered & json parsing fails.then it should return null. 
if you want i can remove the whole test case .